lib/dataspec/src/lib.rs

```diff
@@ -64,6 +64,10 @@ pub struct ClientStats {
 pub struct PubsubStats {
     pub publishers: Publishers,
     pub subscribers: Subscribers,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Do we also want other stats like ok, corrupt, invalid or is just the count and latency sufficient?

Reply:
We should add the rest of the publisher and subscriber stats to this. I'll leave it for another PR though.